docs/data/material/getting-started/templates/sign-in/SignIn.tsx

```diff
@@ -39,73 +111,266 @@ export default function SignIn() {
-          }}
+    <ThemeProvider theme={showCustomTheme ? SignInTheme : defaultTheme}>
+      <CssBaseline />
+      <Stack
+        direction="column"
+        justifyContent="space-between"
+        sx={(theme) => ({
+          backgroundImage:
+            theme.palette.mode === 'light'
+              ? `linear-gradient(180deg, ${alpha('#CEE5FD', 0.2)}, #FFF)`
+              : `linear-gradient(${alpha('#02294F', 0.2)}, ${alpha('#021F3B', 0.0)})`,
+          backgroundRepeat: 'no-repeat',
+          px: { xs: 2, sm: 4 },
+          height: '100dvh',
+        })}
+        component="main"
+      >
+        <Stack direction="row" justifyContent="space-between" sx={{ my: 2 }}>
+          <Button
+            startIcon={<ArrowBackRoundedIcon />}
+            component="a"
+            href="/material-ui/getting-started/templates/"
+          >
+            Back
+          </Button>
+          <ToggleColorMode mode={mode} toggleColorMode={toggleColorMode} />
+        </Stack>
+        <Stack
+          justifyContent={{ xs: 'start', sm: 'center' }}
+          sx={{ height: '100%', mt: { xs: 0, sm: -72 } }}
         >
-          <Avatar sx={{ m: 1, bgcolor: 'secondary.main' }}>
-            <LockOutlinedIcon />
-          </Avatar>
-          <Typography component="h1" variant="h5">
-            Sign in
-          </Typography>
-          <Box component="form" onSubmit={handleSubmit} noValidate sx={{ mt: 1 }}>
-            <TextField
-              margin="normal"
-              required
-              fullWidth
-              id="email"
-              label="Email Address"
-              name="email"
-              autoComplete="email"
-              autoFocus
-            />
-            <TextField
-              margin="normal"
-              required
-              fullWidth
-              name="password"
-              label="Password"
-              type="password"
-              id="password"
-              autoComplete="current-password"
+          <Card
+            variant="outlined"
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              width: { sm: '100%', md: '450px' },
+              mx: 'auto',
+              p: 4,
+              gap: 4,
+            }}
+          >
+            <img
+              src={
+                'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+              }
+              style={logoStyle}
+              alt="Sitemark's logo"
             />
-            <FormControlLabel
-              control={<Checkbox value="remember" color="primary" />}
-              label="Remember me"
-            />
-            <Button
-              type="submit"
-              fullWidth
-              variant="contained"
-              sx={{ mt: 3, mb: 2 }}
+            <Typography
+              component="h1"
+              variant="h4"
+              sx={{ width: '100%', fontSize: 'clamp(2rem, 10vw, 2.5rem)' }}
```

Comment:
```suggestion
              sx={{ width: '100%', fontSize: 'clamp(2rem, 10vw, 2.15rem)' }}
```

I'd reduce this a bit, it's too big